x/ccv/provider/keeper/throttle_test.go

```diff
@@ -895,7 +897,7 @@ func TestPanicIfTooMuchThrottledPacketData(t *testing.T) {
 
 		// Queue packet data instances until we reach the max (some slash packets, some VSC matured packets)
 		reachedMax := false
-		for i := 0; i < int(tc.max+2); i++ { // iterate up to tc.max+1
+		for i := 0; i < int(tc.max); i++ {
```

Comment:
New iteration follows more common conventions, old iteration was kind of confusing